### PR TITLE
Make the recently added Action actually work

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: .github/workflows/deploy-pages.yml [hunk 1]
```diff
@@ -18,7 +18,8 @@ jobs:
       - name: Hugo Deploy GitHub Pages
         uses: benmatselby/hugo-deploy-gh-pages@master
         env:
+          CNAME: packit.dev
+          GITHUB_ACTOR: octocat
```
Comment (TomasTomecek) [inline .github/workflows/deploy-pages.yml:22]: tried to figure out who this is, but couldn't

Comment (jpopelka) [reply .github/workflows/deploy-pages.yml:22]: yeah, and if you checked [the commit message](https://github.com/packit/packit.dev/pull/181/commits/bdbc4fead936f8f7539636e968297aa2c2244e2a) you'd see:

" It's required by the [hugo-deploy-gh-pages Action](https://github.com/benmatselby/hugo-deploy-gh-pages/blob/master/action.sh#L61)
Frankly, I don't know what to put in there and the [docs](https://github.com/marketplace/actions/hugo-deploy-github-pages#environment-variables) doesn't help much."

If you have a better actor, let me know.

Comment (jpopelka) [reply .github/workflows/deploy-pages.yml]: Ok, added myself there instead.

Comment (TomasTomecek) [reply .github/workflows/deploy-pages.yml]: thanks, always forget about commit messages since github is making it "hard" to see them; makes sense!

**Review by TomasTomecek — APPROVED** [review]

**Pull request merged**